packages/venia-ui/lib/components/Accordion/section.js

```diff
@@ -19,7 +19,7 @@ const Section = props => {
 
     const isOpen = openSectionIds.has(id);
     const titleIconSrc = isOpen ? ArrowUp : ArrowDown;
-    const titleIcon = <Icon src={titleIconSrc} />;
+    const titleIcon = <Icon src={titleIconSrc} size={24} />;
```

Comment:
Looks like the default is 24 anyways, at least per the snaps. I do like being explicit though 👍